src/main/java/org/alfresco/repo/site/SiteMembersCannedQuery.java

```diff
@@ -97,29 +93,37 @@ protected boolean isApplyPostQuerySorting()
     {
     	private String siteShortName;
     	private SiteInfo siteInfo;
-    	private Set<SiteMembership> siteMembers;
+    	private Set<SiteUserMembership> siteMembers;
 
     	CQSiteMembersCallback(String siteShortName, List<Pair<? extends Object, SortOrder>> sortPairs)
     	{
     		this.siteShortName = siteShortName;
 			this.siteInfo = siteService.getSite(siteShortName);
-    		this.siteMembers = sortPairs != null && sortPairs.size() > 0 ? new TreeSet<SiteMembership>(new SiteMembershipComparator(sortPairs, SiteMembershipComparator.Type.MEMBERS)) : new HashSet<SiteMembership>();
+    		this.siteMembers = sortPairs != null && sortPairs.size() > 0
+					? new TreeSet<SiteUserMembership>(SiteUserMembership.getComparator(sortPairs))
+					: new HashSet<SiteUserMembership>();
     	}
 
+		/*
+		 * deprecated from 7.0.0

```

Comment:
Is there a reason this is not specified using the JavaDoc annotation "@deprecated"?

Same applies to several other files.